programs/fuzz/fuzz_privkey.c

```diff
@@ -29,7 +29,9 @@ int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
             mbedtls_mpi_init( &DQ ); mbedtls_mpi_init( &QP );
 
             rsa = mbedtls_pk_rsa( pk );
-            mbedtls_rsa_export( rsa, &N, &P, &Q, &D, &E );
+            if ( mbedtls_rsa_export( rsa, &N, &P, &Q, &D, &E ) != 0 ) {
+                abort();
+            }
             mbedtls_rsa_export_crt( rsa, &DP, &DQ, &QP );
```

Comment:
Can you please do the same for `mbedtls_rsa_export_crt`?

Reply:
Ok